packages/cli/src/__snapshots__/run.test.ts.snap

```diff
@@ -51,17 +51,33 @@ exports[`Run command should execute "package-three start" and exit with 1: stder
 "
 `;
 
+exports[`Run command should execute "package-three start" and exit with 1: stderr 2`] = `
```

Comment:
Hmmm, the changes in those snapshots are a little bit surprising as the CLI tests are untouched. Do you happen to know where they are coming from?

Reply:
This should be fixed in the latest push.

Note that I was having a lot of trouble running these tests locally -- it turns out the bootup time of loading the babel "inline" transpiler to run the cli can take exactly on the edge of 5 seconds, which is the default unit test timeout. (I bumped the timeout to 10 seconds for the cli test and they are a lot more stable on my MacBook now.)